src/io.rs

```diff
@@ -24,10 +24,18 @@ mod writer;
 pub(crate) mod der_writer;
 
 /// A serialized positive integer.
-#[derive(Copy, Clone)]
+#[derive(Copy, Clone, Debug)]
```

Comment:
We intentionally don't derive `Debug` for this type because it contains/references potentially-secret data. We should document this policy in a comment.